pom.xml

```diff
@@ -287,7 +288,7 @@
                     </reporting>
                 </configuration>
             </plugin>
-            <plugin>
+<!--            <plugin>
```

Comment:
1. Any reason why this is being commented instead of removing?
2. What about the bundle symbolic name (and extension name), when will that change?

Reply:
> gpg plugin is needed for releasing, not needed for building. I commented it out temporary to allow passing builds. I see that someone else will need to move it to the proper release profile later.
> 

OK, then fine. 

> I don't think that it has to be changed. We are changing Maven coordinates only. Correct me if I'm wrong.

Yes, I understand just the artifact coordinates are being changed. In general practice, bundle symbolic name is <groupId>.<artifactId>. So I was just curious if bundle symbolic name is also subject to change in future.